### ISSUE TITLE
[UI] Improving the background and Card style of Dummygram

### ISSUE THREAD

**Issue by Frs2304** [narayan954/dummygram#213]
### What would you like to share?

Description:
Currently, the background and card style of Dummygram could use some improvement. The plain background and simple card design do not add much visual interest to the app. I suggest that we explore some new options for the background and card style to make Dummygram more visually appealing.
Some potential ideas for the background could include using a subtle gradient or some image. For the card style, we could experiment with rounded corners, drop shadows, or different border styles to give the cards a more polished look.
Overall, I believe that these improvements could make Dummygram more attractive to users and help it stand out from other similar apps. I look forward to work on this issue and collaborating on potential solutions. @narayan954

**Comment by Frs2304** [comment]
@narayan954 Yes I saw the #4 UI issue only after that I had created this issue. Yeah I want to improve the background and card style in this issue. I want to participate through `JWOC` and please add `medium` label to this issue.

**Comment by Frs2304** [comment]
Hey! @narayan954 I am working on this issue. Just found that some CSS background-color properties are not working correctly. I am working closely to solve this issue and will probably make the PR until today.

**Comment by narayan954** [comment]
> Hey! @narayan954 I am working on this issue. Just found that some CSS background-color properties are not working correctly. I am working closely to solve this issue and will probably make the PR until today.

Okay Tushar, best of luck!

**Comment by Code1705** [comment]
I want to contribute to this issue , please assign me this issue under JWOC

**Comment by Frs2304** [comment]
> I want to contribute to this issue , please assign me this issue under JWOC
> 

Hey I am working on this issue.

**Comment by narayan954** [comment]
> > I want to contribute to this issue , please assign me this issue under JWOC
> 
> Hey I am working on this issue.

right, and @Code1705 you're assigned another issue as well.

**Comment by narayan954** [comment]
closing as stale
